prometheus-metrics/src/main/scala/org/http4s/metrics/prometheus/Prometheus.scala

```diff
@@ -78,18 +97,44 @@ object Prometheus {
   def collectorRegistry[F[_]](implicit F: Sync[F]): Resource[F, CollectorRegistry] =
     Resource.make(F.delay(new CollectorRegistry()))(cr => F.delay(cr.clear()))
 
-  /** Creates a  [[MetricsOps]] that supports Prometheus metrics
-    * *
-    * * @param registry a metrics collector registry
-    * * @param prefix a prefix that will be added to all metrics
+  /** Creates a [[MetricsOps]] that supports Prometheus metrics
+    *
+    * @param registry a metrics collector registry
+    * @param prefix a prefix that will be added to all metrics
+    * @param metricsNames names of the metrics that will be used as a suffix for the metrics
     */
   def metricsOps[F[_]: Sync](
```

Comment:
NIT: I'd personally avoid the overloaded method name here. I realize it is more verbose to have something like,

```scala
def metricsOpsWithNames[F[_]: Sync](...)
```

but I think it is more clear to the users.

Reply:
I've chosen the same name just for avoiding verbose name. But I'm also not sure, maybe verbosity here not so bad... We need one more opinion on this, I suppose.